tests/unit/scanners/test.javascript.js

```diff
@@ -323,8 +261,8 @@ describe('JavaScript Scanner', () => {
 
     const jsScanner = new JavaScriptScanner('', 'badcode.js');
 
-    await jsScanner.scan();
-    expect(jsScanner._rulesProcessed).toEqual(Object.keys(rules).length);
+    await runJsScanner(jsScanner);
+    expect(jsScanner._rulesProcessed).toEqual(16);
```

Comment:
it would be nice it we could get the expected value for jsScanner._rulesProcessed in a programmatic way instead of using directly the value the we currently expect.

if there isn't a way to avoid this, it may be good to leave a comment to explain where that value is coming from and how a new expected value should be determined if it has to be updated in the future.

Reply:
I added a comment.